src/demos/FeatureSelectDemo.vue

```diff
@@ -0,0 +1,191 @@
+<template>
+  <div>
+    <label for="bufferRadius"> Buffer Radius: </label>
+    <input type="number" id="bufferRadius" v-model="bufferRadius" />
+  </div>
+
+  <ol-map
+    :loadTilesWhileAnimating="true"
+    :loadTilesWhileInteracting="true"
+    style="height: 400px"
+    @pointermove="pointermove"
+    @click="click"
+    ref="mapRef"
+  >
+    <ol-view ref="view" :center="center" :rotation="rotation" :zoom="zoom" />
+
+    <ol-vector-tile-layer>
+      <ol-source-vector-tile :url="url" :format="mvtFormat">
+      </ol-source-vector-tile>
+
+      <ol-style>
+        <ol-style-stroke color="blue" :width="1"></ol-style-stroke>
+      </ol-style>
+    </ol-vector-tile-layer>
+
+    <ol-vector-layer>
+      <ol-source-vector :features="highlightedFeature"> </ol-source-vector>
+
+      <ol-style>
+        <ol-style-stroke color="red" :width="3"></ol-style-stroke>
+      </ol-style>
+    </ol-vector-layer>
+
+    <ol-vector-layer>
+      <ol-source-vector :features="selectedFeatures"> </ol-source-vector>
+
+      <ol-style>
+        <ol-style-stroke color="red" :width="3"></ol-style-stroke>
+      </ol-style>
+    </ol-vector-layer>
+
+    <ol-vector-layer>
+      <ol-source-vector :features="bound"> </ol-source-vector>
+
+      <ol-style>
+        <ol-style-stroke color="green" :width="3"></ol-style-stroke>
+      </ol-style>
+    </ol-vector-layer>
+  </ol-map>
+</template>
+
+<script setup lang="ts">
+import * as turf from "@turf/turf";
+import Feature, { type FeatureLike } from "ol/Feature";
+import type MapRef from "ol/Map.js";
+import { Geometry } from "ol/geom";
+import GeoJSON from "ol/format/GeoJSON";
+import VectorTileLayer from "ol/layer/VectorTile";
+import { transform } from "ol/proj";
+import { inject, ref, watch } from "vue";
+
+const center = ref([943955.94569529717, 6356667.343082143]);
+const zoom = ref(18);
+const rotation = ref(0);
+
+const url = ref(
+  "https://basemaps.arcgis.com/arcgis/rest/services/World_Basemap_v2/VectorTileServer/tile/{z}/{y}/{x}.pbf",
+);
+const format = inject("ol-format");
+const mvtFormat = new format.MVT({ featureClass: Feature });
+
+const mapRef = ref<{ map: MapRef } | null>(null);
+
+let selectedFeatures = ref<FeatureLike[]>([]);
+let highlightedFeature = ref<FeatureLike[]>([]);
+let bound = ref<FeatureLike[]>([]);
+let bufferRadius = ref<number>(10);
+let highlightingTemplate = ref<number[][][]>([]);
+
+function pointermove(event: any) {
+  const map = mapRef.value?.map;
+
+  highlightedFeature.value = [];
+
+  if (map) {
+    map.forEachFeatureAtPixel(
+      event.pixel,
+      (feature: FeatureLike) => {
+        highlightedFeature.value = [feature];
+      },
+      {
+        hitTolerance: 1,
+      },
+    );
+  }
+}
+
+function reMap(espg3857: number[][]): number[][] {
```

Comment:
I think something like `transformCoordinatess` would be a better name, wouldn't it?